test/WebJobs.Script.Tests.Integration/SamplesEndToEndRestartTests.cs

```diff
@@ -1,79 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Net.Http;
-using System.Threading.Tasks;
-using Newtonsoft.Json.Linq;
-using Xunit;
-
-namespace Microsoft.Azure.WebJobs.Script.Tests
-{
-    // Running these in their own class with private fixtures so they do not affect the timing of
-    // subsequent tests.
-    public class SamplesEndToEndRestartTests
-    {
-        [Fact(Skip = "Will not pass until https://github.com/Azure/azure-webjobs-sdk-script/issues/1537 is fixed")]
```

Comment:
I think the test will be hard to stabilize, but I agree with you in principal. My goal here was to fix the actual specific bug the issue pointed to and unit test that sufficiently. For this fix, unit test is better.